joomla/joomla-cms
# [4.0] Re-add archive in the frontend

Pull Request for Issue #21533 .

### Summary of Changes
Read the archived functionallity in the frontend


### Testing Instructions
Create an archived link and the archived module
Check and compare the functionality with J! 3
